docs/development/minimal-reproductions.md

```diff
@@ -1,49 +1,80 @@
 # About minimal reproductions
 
-You may be asked to provide a "minimal reproduction" repository to help the Renovate team debug bug reports or work on a feature.
+We may ask you to create a "minimal reproduction" repository to help us fix bugs or work on a feature.
 
-This document explains why we need a minimal reproduction, why using large repositories to debug is undesirable, and how you can create a good minimal reproduction.
+This document explains why we need a minimal reproduction, why we won't use your production repository to debug, and how to create a good minimal reproduction.
+
+## Help yourself by creating a minimal reproduction
+
+A minimal reproduction helps us to:
+
+- Fix your bug faster
+- Work on your feature request faster
+- Confirm the cause of the behavior
+
+It's fastest if you - as the bug reporter or feature requester - create the reproduction.
+
+## How the Renovate developers use your minimal reproduction
+
+TODO: `@rarkins` should fill out this section.
```

Comment:
```suggestion
The first benefit of a public reproduction is to prove that the problem is not caused by your environment or by a setting you left out of your description, thinking it wasn't relevant.
If there were any doubts about whether you'd found a genuine problem before, they are usually removed once a reproduction is made.

Next, when a reproduction has minimal config, it can often let us narrow down or even identify the root cause, suggest workarounds, etc.
This means we can often help you from code inspection alone.

Finally, by making the code/dependencies minimal, it usually makes the problem feasible to step through using a debugging if code inspection wasn't sufficient.
Production repositories or non-minimal reproductions are often very difficult to debug because break points get triggered dozens or hundreds or times.
```